integration_tests/grid_fee.feature

```diff
@@ -109,7 +109,7 @@ Feature: GridFee integration tests
      And d3a uses an two-sided pay-as-clear market
      And the minimum offer age is <min_offer_age>
      And d3a dispatches events from top to bottom
-     When we run the simulation with setup file grid_fees.constant_grid_fees and parameters [24, 60, 60, 0, 1]
+     When we run the simulation with setup file grid_fees.constant_grid_fees and parameters [24, 60, 60, 1]
```

Comment:
Well, no harm having it here, but since most of the tests use the percentage fees, why not having a default value of percentage fees (without changing this parameter list) and if there is a need for a constant fee in a test, only then to add an extra parameter ? Since you did it this way it's not important now, just that you could have avoided the hassle of changing all the integration tests with this approach.

Reply:
Hmm i dont understand. There is no global grid_fee setting any more. The grid fees are set in the setup file. So i deleted this parameter. 
Or can you rephrase please?

Reply:
I see, I misread the parameters here. So the last parameter is always the market_count, right ? Then all is good.